### ISSUE TITLE
Cannot read property 'Utils' of undefined

### ISSUE THREAD

**Issue by myartsev** [shipshapecode/ember-drop#67]
Running `ember install ember-drop` on a brand new Ember 2.13 project (and not even doing anything with it yet) crashes loading any route with the following error:

```
drop.min.js:1 Uncaught TypeError: Cannot read property 'Utils' of undefined
    at drop.min.js:1
    at drop.min.js:1
    at drop.min.js:1
(anonymous) @ drop.min.js:1
(anonymous) @ drop.min.js:1
(anonymous) @ drop.min.js:1


loader.js:232 Uncaught Error: Could not find module `ember-resolver` imported from `drop-test/resolver`
    at missingModule (loader.js:232)
    at findModule (loader.js:243)
    at Module.findDeps (loader.js:173)
    at findModule (loader.js:247)
    at Module.findDeps (loader.js:173)
    at findModule (loader.js:247)
    at requireModule (loader.js:26)
    at app-boot.js:2
missingModule @ loader.js:232
findModule @ loader.js:243
Module.findDeps @ loader.js:173
findModule @ loader.js:247
Module.findDeps @ loader.js:173
findModule @ loader.js:247
requireModule @ loader.js:26
(anonymous) @ app-boot.js:2
```

Don't know if this is specific to Ember 2.13 or if it was happening with earlier versions as well?

**Comment by rwwagner90** [comment]
Unsure what the issue is. I just updated a bunch of deps and released 1.2.1. Can you please see if the issue still persists?

**Comment by myartsev** [comment]
Still there :S

Do you not get the same issue?
`ember new drop-test`  
`cd drop-test`  
`ember install ember-drop`  
`ember s`  

That is all it takes for me.

**Comment by rwwagner90** [comment]
@myartsev it seems that `ember-tether` needs to be installed. You can manually `ember install ember-tether` or I just released version 1.2.2 which has a blueprint that add it for you, if you just run `ember install ember-drop`. Sorry about that! I had thought `ember-tether` fixed this issue, but I was wrong.
